themes/default/content/what-is/what-is-secrets-management.md

```diff
@@ -0,0 +1,73 @@
+---
+title: What Is Secrets Management?
+meta_desc: |
+    Understand what is infrastructure as code, along with the main benefits and importance
+    for modern application development.
+
+type: what-is
+page_title: What is secrets management?
+
+customer_logos:
+  title: Leading engineering organizations are building with Pulumi
+  logos:
+    - items:
+      - snowflake
+      - tableau
+      - atlassian
+      - fauna
+      - ware2go
+    - items:
+      - mindbody
+      - sourcegraph
+      - fenergo
+      - skai
+      - lemonade
+    - items:
+      - clearsale
+      - angellist
+      - webflow
+      - supabase
+      - ro
+---
+
+In the vast digital landscape of today's interconnected world, the security of sensitive information is of utmost importance. Imagine a world where personal data, passwords, and confidential details are guarded like treasures, shielded from prying eyes and potential threats. This is where secrets management comes into play – an essential aspect of safeguarding valuable information in the digital realm.
```

Comment:
Not sure if its here or needs a new section but we should dive into a bit more about sprawl of sensitive data and not trying to get breaches. Like what is the pain point that necessitated secrets management